README.md

```diff
@@ -119,7 +119,7 @@ The process SIG Interoperability follows can be seen from [here](https://github.
 Chairs and the TOC Sponsor of the SIG are
 
 * Melissa McKay ([@mjmckay](https://github.com/mjmckay)), JFrog - Chair
-* Kara de la Marck ([@MarckK](https://github.com/MarckK)), Continuous Delivery Foundation - Chair
+* Dadisi Sanyika([@dsanyika](https://github.com/dsanyika)), Apple - Chair
```

Comment:
Shall we start an emeritus list?

Reply:
That's a good idea. Let me send a new commit and add a section similar to the one in TOC readme.